html/hr/demos/hr.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+  <title>Горизонтальная линия — <hr> — Дока</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="preconnect" href="https://fonts.gstatic.com/" crossorigin>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;1,400;1,500;1,900&display=swap">
+  <style>
+    .inline {
+      padding: 5px 10px;
+      font-size: 18px;
+      font-weight: 500;
+    }
```

Comment:
Шрифт мало подключить, его надо использовать :) Ну и если использовать, то лучше вынести всё как в типичном проекте на корневой элемент, который всё отнаследует.

```suggestion
  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto&display=swap">
  <style>
    body {
      font-family: 'Roboto', sans-serif;
      font-size: 18px;
    }

    .inline {
      padding: 5px 10px;
    }
```

Reply:
Кажется, надо переставать писать статьи по ночам... Немного не так донесла мысль